nazi_zombies_portable/README.md

```diff
@@ -0,0 +1,11 @@
+# Nazi Zombies: Portable (NZ:P)
+
+Nazi Zombies: Portable is a project that allows you to play Call of Duty: Zombies on a LOT of devices/platforms, from Windows to Linux to Browser to Nintendo Switch to PSP to PlayStation VITA (you get the idea).
+It is based on the FTEQW engine and this egg can be used as a plain FTEQW egg if you desire.
+
+## Port
+27500 TCP (for WS connection for Web Clients), 27500 UDP (for Native Clients) - Port for both TCP and UDP can be changed by modifying `sv_port_tcp` and `sv_port` variables respectively.
```

Comment:
Please use the table format, as all eggs use.

And {{server.port}} comes from the primary allocation so no need to tell how to change that, it will auto happen if the primary allocation is changed

Reply:
Ah, yeah, I saw one that used the table but it had multiple ports, table seems overkill for 1 port but I can do it

Oh yeah, I guess I was being too technical, did just want to state the variables specificially as it took me a bit to figure out but I did also make a dedicated server page for the nzp docs so I guess I can shorten it, with the table  I will also probably divy by TCP and UDP as they are fundamentally different (wish I knew what UDP it used, they say Client Side Quake Protocol is only supported value in cvar description but why would it be client side quake if it is a dedicated server. but anyway, will turn into a table. split the port into 27500 TCP and UDP, 2 entries, describing the TCP is for WS/Web Clients while UDP is for native (as I think that is good info to have, if you don't care about web clients, you can just only allow udp traffic for the port in the firewall and vise versa)